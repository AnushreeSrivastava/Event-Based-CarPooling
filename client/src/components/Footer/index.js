import React from "react";
import Navbar from "react-bootstrap/Navbar";
import Nav from "react-bootstrap/Nav";
import styled from "styled-components";

const Styles = styled.div`
  .content {
    margin-left: 40%;
  }
`;

const Footer = props => {
  return (
    <Styles>
      <Navbar bg="dark" expand="lg" variant="dark">
        <Nav.Link href="/" className="content">
          @EBCSystems
        </Nav.Link>
        <Nav.Link href="/">About</Nav.Link>
        <Nav.Link href="/">Contact</Nav.Link>
      </Navbar>
    </Styles>
  );
};

export default Footer;
