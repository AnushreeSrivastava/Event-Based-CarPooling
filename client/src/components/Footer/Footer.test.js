import React from 'react';
import Footer from "../Footer";
import { mount, shallow, render } from 'enzyme';
import Nav from "react-bootstrap/Nav";

const clickFn = jest.fn();

describe('Footer Component testing', () => {
  it('should render Footer component', () => {
    const component = shallow(<Footer/>);
    expect(component).toMatchSnapshot();
  });

  it('should have about, contact and Logo links', () => {
    const wrapper = mount(<Footer/>);
    const about = <Nav.Link href="/">About</Nav.Link>;
    const contact = <Nav.Link href="/">Contact</Nav.Link>;
    const logo = <Nav.Link href="/" className="content">@EBCSystems</Nav.Link>;

    expect(wrapper.contains(about)).toEqual(true);
    expect(wrapper.contains(contact)).toEqual(true);
    expect(wrapper.contains(logo)).toEqual(true);    
  });  
});
