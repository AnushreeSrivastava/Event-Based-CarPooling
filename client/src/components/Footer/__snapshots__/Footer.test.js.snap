// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component testing should render Footer component 1`] = `ShallowWrapper {}`;
